custom-vu/src/main/kotlin/jces1209/vu/action/LogInWithAtlassianId.kt

```diff
@@ -38,6 +39,7 @@ class LogInWithAtlassianId(
 
     private fun logIn() {
         jira.goToLogin()
+        Thread.sleep(Random().nextInt(300000)+300000L)
```

Comment:
formatting nitpick: usually we'd want spaces around the `+` operator (at least in Java Style Guide, assuming same applies to Kotlin)

Reply:
Am I right to interpret this as sleep randomly from 5 to 10 minutes? That would probably be too long, we know the limit is no more than 20 attempts within 2 minutes. So think if we could have a shorter lower bound, it would be good:

```suggestion
        Thread.sleep(Random().nextInt(180000) + 120000L) // Sleep randomly from 2 -> 5 minutes
```

Reply:
It's better to not create a new `Random` generator for every time you want a random number. This is true in Java and Kotlin. In Kotlin using the `Random` companion object saves you from even having to create a generator, [according to the docs](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.random/-random/).

Reply:
The constant duration expressions could be made more intuitive by using `Duration.ofMinutes(2).toMillis()`, no comments required :-)

Reply:
Ben, we are using an older version of Kotlin and needs kotlin higher than 1.3 for this :-( 
other comments actioned.
Thanks